source/retryable-writes/retryable-writes.rst

```diff
@@ -66,7 +66,23 @@ ClientSession
    name of this object MAY vary across drivers.
 
 Retryable Error
-   An error is considered retryable if it meets any of the following criteria:
+   An error is considered retryable if it has a RetryableWriteError label in
+   its top-level "errorLabels" field.
+
+   For server versions 4.3 and newer, MongoDB will add a RetryableWriteError label to
```

Comment:
I think we should either refer to the specific 4.3.x release (if possible), or stick to stable release versions and use "4.4". I believe the convention across most specs is to just use the stable version.

It's also not a problem if the spec ends up saying "4.4" and we use a specific 4.3.x version in the test file requirements.

Reply:
Good to know, thanks.